src/api/axios.ts

```diff
@@ -0,0 +1,49 @@
+import axios from "axios";
+
+export interface QueryParams {
+  [key: string]: string;
+}
+
+/* 
+    axios use
+
+    const endpoint = "/endpoint"
+    ***pass query params in an object as name:value pairs
+    await axiosRequest({populate: "*", sort: "asc"}).get(endpoint) 
+      .catch((err: any) => handle error)
+
+    The request url will look like this: http://baseurl.com/endpoint?populate=*&sort=asc
```

Comment:
The url would look like: 

GET: http://baseurl.com/endpoint?populate=*&sort=asc